lib/project_types/script/layers/application/create_script.rb

```diff
@@ -31,22 +31,17 @@ def create_project(ctx, script_name, extension_point)
             ScriptProject.current
           end
 
-          def install_dependencies(ctx, language, script_name, extension_point, project)
+          def install_dependencies(ctx, language, script_name, project, project_creator)
```

Comment:
My plan after this is to make `project_creator` and `task_runner` available on the project, which will result in cleaning up this file and method interface. I just wanted to break up the code so that the PRs aren’t massive and doing multiple things